en/control-structures.md

```diff
@@ -3,13 +3,12 @@ layout: default
 language: 'en'
 version: '0.10'
 ---
+
 # Control Structures
 Zephir implements a simplified set of control structures present in similar languages like C, PHP etc.
 
-<a name='conditionals'></a>
 ## Conditionals
 
-<a name='conditionals-if'></a>
 ### If Statement
```

Comment:
@Arhell This will change link to this page. Because with this change this will rendrered as `<a name='if-statement'></a>` . This will cause you to find all the pages that could link to this section and update them (and each time when someone will rename header). Can we just change this to use `id` instead of `name`?

Reply:
Ok, I want to test